man/systemd-nspawn.xml

```diff
@@ -336,6 +336,21 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>--pivot-root=</option></term>
+
+        <listitem><para>Pivot the specified directory to <filename>/</filename> inside the container, and either remove the
+        container's old root, or pivot it to another specified directory. Takes one of: a path argument — in which case the
+        specified path will be pivoted to <filename>/</filename> and the old root will be removed; or a colon-separated tuple
+        of new root path and pivot destination for the old root. The new root path will be pivoted to <filename>/</filename>,
+        and the old <filename>/</filename> will be pivoted to the other directory. Both paths must be absolute, and are resolved
+        in the container's file system namespace.</para>
+
+        <para>This is for containers which have several bootable sysroots in them; for example, several OSTree deployments. It
+        emulates the behavior of the boot loader and initramfs which normally select which sysroot to mount as root and start
```

Comment:
please write "initial RAM disk" or so instead of "initramfs". Similar "sysroot" isn't really a word, is it? Maybe just call this "directory"...